docs/pages/build-reference/infrastructure.md

```diff
@@ -2,45 +2,48 @@
 title: Build server infrastructure
 ---
 
-This document describes the current build infrastructure as of December 11, 2020. It is likely to change over time, and this document will be updated.
+This document describes the current build infrastructure as of December 15, 2020. It is likely to change over time, and this document will be updated.
 
 The software components will become customizable, but they aren't yet. So there is only one version of Node, yarn, CocoaPods, Xcode, Ruby, Fastlane, and so on currently available.
 
 ## JavaScript environment
 
-- node 14.15.1
-- yarn 1.22.10
+- Node.js 14.15.1
+- Yarn 1.22.10
 
 ## Android build server configuration
 
-- one pod per k8s node (gcp n2-standard-2)
-  - 2 CPU (gcp unit)
-  - 8 GB RAM (with a bit over 6 GB available for build)
-- docker image: `ubuntu:bionic-20201119`
-- NDK 19.2.5345600
-- npm cache deployed inside k8s cluster
-- maven cache deployed inside k8s cluster
-    - `maven-central` - [https://repo1.maven.org/maven2/](https://repo1.maven.org/maven2/)
-    - `google` - [https://maven.google.com/](https://maven.google.com/)
-    - `android-tools` - [https://dl.bintray.com/android/android-tools/](https://dl.bintray.com/android/android-tools/)
-    - `jcenter` - [https://jcenter.bintray.com/](https://jcenter.bintray.com/)
-- global `gradle.properties`:
-
-```jsx
-org.gradle.jvmargs=-Xmx6g -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
-org.gradle.parallel=true
-org.gradle.configureondemand=true
-org.gradle.daemon=false
-```
+- Android workers run on Kubernetes in an isolated environment
+  - Every build gets its own container running on a dedicated Kubernetes node
+  - Build resource limits: 3 CPU, 12 GB RAM
+- Installed software:
+  - Docker image: `ubuntu:bionic-20201119`
+  - NDK 19.2.5345600
+- NPM cache deployed inside k8s cluster
+- Maven cache deployed inside k8s cluster
+  - `maven-central` - [https://repo1.maven.org/maven2/](https://repo1.maven.org/maven2/)
+  - `google` - [https://maven.google.com/](https://maven.google.com/)
+  - `android-tools` - [https://dl.bintray.com/android/android-tools/](https://dl.bintray.com/android/android-tools/)
+  - `jcenter` - [https://jcenter.bintray.com/](https://jcenter.bintray.com/)
```

Comment:
those 3 repositories and yet cached, not sure if we should mark them or comment them